changelog.d/20768-default_gelf_stream_framing.breaking.md

```diff
@@ -0,0 +1,31 @@
+Now the GELF codec with stream-based sources uses null byte (`\0`) by default as messages delimiter instead of newline (`\n`) character.
+
+### Configuration changes
+
+In order to maintain the previous behavior, you must set the `framing.method` option to the `character_delimited` method and the `framing.character_delimited.delimiter` option to `\n` when using GELF codec with stream-based sources.
+
+### Example configuration change for socket source
```

Comment:
Awesome, thanks for including the example for using the previous default.

Small nit that we try to use YAML rather than TOML as the default these days.

Reply:
Done!